snap/info_snap_yaml.go

```diff
@@ -414,8 +414,8 @@ func convertToSlotOrPlugData(plugOrSlot, name string, data interface{}) (iface,
 	}
 }
 
-// validateAttr validates an attribute value and returns a normalized version of it (map[interface{}]interface{} is turned into map[string]interface{})
-func validateAttr(v interface{}) (interface{}, error) {
+// normalizeYamlValue validates values and returns a normalized version of it (map[interface{}]interface{} is turned into map[string]interface{})
+func normalizeYamlValue(v interface{}) (interface{}, error) {
```

Comment:
I cannot think of a concrete use case, but I think it won't harm to have floats supported. Added.